docs/menus/oscillator/wave_index.md

```diff
@@ -0,0 +1,3 @@
+Index into the wavetable.
+
+This is in the standard 0-50 range, but modulation can reach values between the increments presented by this menu item.
```

Comment:
There's also some hacks to make sure that modulation with an LFO can always access the full range of the wavetable with depth 50